tests/arg_parse/test_arg_parse_version_check.cpp

```diff
@@ -0,0 +1,75 @@
+// ==========================================================================
+//                                 arg_parse
+// ==========================================================================
```

Comment:
Please use the following header:
// ==========================================================================
//                 SeqAn - The Library for Sequence Analysis
// ==========================================================================